packages/fxa-admin-panel/src/components/AccountSearch/index.test.tsx

```diff
@@ -141,18 +141,158 @@ function exampleBounceMutationResponse(email: string): MockedResponse {
   };
 }
 
+const MinimalAccountResponse = (testEmail: string) => ({
+  data: {
+    accountByEmail: {
+      uid: '123',
+      createdAt: 1658534643990,
+      disabledAt: null,
+      lockedAt: null,
+      emails: [
+        {
+          email: testEmail,
+          isVerified: true,
+          isPrimary: true,
+          createdAt: 1658534643990,
+        },
+      ],
+      emailBounces: [],
+      securityEvents: [],
+      totp: [],
+      recoveryKeys: [],
+      linkedAccounts: [],
+      attachedClients: [],
+      subscriptions: [],
+    },
+  },
+});
+
+const nextTick = (t?: number) => new Promise((r) => setTimeout(r, t || 0));
```

Comment:
nit: `t` isn't used in any of the calls so no need to have that param?

Reply:
It is in fact. Check the setTimeout call.

Reply:
Ah, I gotcha, you mean the calling code never uses it. Alright, I can remove it.